docs/getting-started.md

```diff
@@ -20,12 +21,12 @@ Before you get started:
 
 > ⚠️ This project uses Go Modules. Due to the current state of code generation in [controller-tools](https://github.com/kubernetes-sigs/controller-tools) and [code-generator](https://github.com/kubernetes/code-generator), KUDO currently **must** be cloned into its `$GOPATH`-based location.
 
-## Installation Instructions
+## Install KUDO to your cluster
 
-- Get KUDO repo: `go get github.com/kudobuilder/kudo/`
+- `mkdir -p $GOPATH/src/github.com/kudobuilder/kudo`
 - `cd $GOPATH/src/github.com/kudobuilder/kudo`
-- `kubectl apply -f config/crds` to deploy the universal CRDs
-- `make run` to run the Operator with local go environment or `make deploy | kubectl apply -f -` to install to your kubernetes cluster
+- Get KUDO repo: `git clone git@github.com:kudobuilder/kudo.git`
```

Comment:
AFAIK with go modules, `go get` does not precisely do what it did on systems prior to go modules, so it no longer fetches code into gopath. Aleksey had problems with that when installing into local env. I believe this is fail-safe steps :)

Reply:
This will mean users end up with `$GOPATH/src/github.com/kudobuilder/kudo/kudo` - we probably want to drop the `/kudo` from the previous two commands.

Reply:
Agreed with @yankcrime, nested one level too deep. :)

Reply:
So change just to `mkdir -p $GOPATH/src/github.com/kudobuilder`

Reply:
actually, they can clone this anywhere, since they're just creating/applying some YML